taxi/forms.py

```diff
@@ -0,0 +1,59 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",
+        )
+
+    def clean_license_number(self):
```

Comment:
Create a mixin class which will emplement license validation login from which you will inherit
Split checks to give users clarity errors